src/test/streams_tests.cpp

```diff
@@ -9,8 +9,7 @@
 #include <boost/assign/std/vector.hpp> // for 'operator+=()'
 #include <boost/assert.hpp>
 #include <boost/test/unit_test.hpp>
-                    
-using namespace std;
+
 using namespace boost::assign; // bring 'operator+=()' into scope
```

Comment:
You mean `boost::assign`? It's not trivial to remove, as it actually adds an `operator` so that things like the `key` statement below are possible.
```C++
    std::vector<unsigned char> key;
    [...]
    key += '\x00','\x00';
```